README.md

```diff
@@ -5,5 +5,12 @@ Repo for Administrator
 1. If you do not have maven installed, you need to install it
 2. Import this project as a maven project
 3. Set up MySQL schema using the schema in src/test/resources/schema.sql
-4. Change root and password in src/main/resources/application.properties
-        to access your MySQL server with the library schema
+4. Create a file called `database-config.properties` in src/main/resources. Look at the code down below for an example. <br/>
+```
+spring.datasource.url = jdbc:mysql://hostname:port/data-base-name?useSSL=false&serverTimezone=UTC
```

Comment:
"Database" is a single word; it looks wrong to have it divided in the sample URL.

Also, I would remove the `useSSL=false` from the URL here, and mention that it is generally needed when connecting to *localhost* elsewhere in the `README`; if a database isn't local, and is properly set up, the user should *want* to make an encrypted connection.